geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-convert/src/main/scala/org/locationtech/geomesa/fs/storage/converter/ConverterStorage.scala

```diff
@@ -27,6 +27,8 @@ class ConverterStorage(fc: FileContext,
                        converter: SimpleFeatureConverter,
                        partitionScheme: PartitionScheme) extends FileSystemStorage {
 
+  // TODO close converter...
```

Comment:
Do we need to address this TODO?  If it isn't stray, can we enter a ticket describing the work a bit more?

Reply:
the problem is that we aggressively cache storage instances, so even if we wired a 'close' method through the entire storage api, we'd also have to implement a 'borrow/return' paradigm and expire idle convictions. Since this only affects redis caches, and since it will only open a single connection now (instead of one per evaluation context), there shouldn't really be an impact to not closing it. i'll update the TODO but am hesitant to add a ticket as i think the solution would be worse than the problem